modules/nodes-sigstore-configure-cluster-policy.adoc

```diff
@@ -0,0 +1,194 @@
++
+[source,terminal]
+----
+$ oc edit featuregate cluster
+----
++
+.Example `FeatureGate` CR
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: TechPreviewNoUpgrade <1>
+----
+<1> Enables the required `SigstoreImageVerification` feature.
++
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. This feature set allows you to enable these Technology Preview features on test clusters, where you can fully test them. Do not enable this feature set on production clusters.
+====
++
+After you save the changes, new machine configs are created, the machine config pools are updated, and scheduling on each node is disabled while the change is being applied.
+	
+.Procedure
+
+. Create a cluster image policy object similar to the following examples. See "About image policy parameters" for specifiec details on these parameters.
++
+--
+.Example cluster image policy object with a public key policy 
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: ClusterImagePolicy <1>
+metadata:
+  name: p1
+spec:
+  scopes: <2>
+    - example.com
+  policy: <3>
+    rootOfTrust: <4>
+      policyType: PublicKey <5>
+      publicKey:
+        keyData: a2V5RGF0YQ== <6>
+        rekorKeyData: cmVrb3JLZXlEYXRh <7>
+    signedIdentity: <8>
+      matchPolicy: MatchRepoDigestOrExact
+----
+<1> Creates a `ClusterImagePolicy` object.
+<2> Defines a list of images scopes assigned to this policy. In a cluster image policy, make sure that the policy does not block the deployment of the {product-title} images in the `quay.io/openshift-release-dev/ocp-release` and `quay.io/openshift-release-dev/ocp-v4.0-art-dev` repositories. Images in these repositories are required for cluster operation..
+<3> Specifies the parameters that define how the images are verified.
+<4> Defines a root of trust for the policy.
+<5> Specifies the policy types that define the root of trust: here, a public key with Rekor verification.
+<6> For a public key policy, specifies a base64-encoded public key in the PEM format. The maximum length is 8192 characters.
+<7> Optional: Specifies a base64-encoded Rekor public key in the PEM format. The maximum length is 8192 characters.
+<8> Specifies one of the following processes to verify the identity in the signature and the actual image identity:
+* `matchRepoDigestOrExact` 
+* `matchRepository`.
+* `exactRepository`
+* `remapIdentity`
+--
++
+--
+.Example cluster image policy object with a Fulcio certificate policy and the `remapIdentity` match policy
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: ClusterImagePolicy <1>
+metadata:
+  name: p1
+spec:
+  scopes: <2>
+    - example.com
+  policy: <3>
+    rootOfTrust: <4>
+      policyType: FulcioCAWithReko <5>
```

Comment:
```
apiVersion: config.openshift.io/v1alpha1
kind: ClusterImagePolicy
metadata:
  name: p1
spec:
  scopes:
    - example.com
  policy:
    rootOfTrust:
      policyType: FulcioCAWithRekor
      fulcioCAWithRekor:
        fulcioCAData: a2V5RGF0YQ==
        fulcioSubject:
          oidcIssuer: https://expected.OIDC.issuer
          signedEmail: expected-signing-user@example.com
        rekorKeyData: cmVrb3JLZXlEYXRh
    signedIdentity:
      matchPolicy: RemapIdentity
      remapIdentity:
        prefix: example.com
        signedPrefix: mirror-example.com
```

This is a valid FulcioCAWithRekor example, and I have also changed the fulcioCAData to valid bae64 encoding data.